### PR TITLE
Support more than two operands in OperatorNode for addition and multiplication

### PR DESCRIPTION
**Pull request by FSMaxB**
See #739

### CODE REVIEW

File: lib/expression/node/OperatorNode.js [hunk 1]
```diff
@@ -269,6 +269,29 @@ function factory (type, config, load, typed, math) {
...
+          parens = false;
+        } else if ((precedence === argPrecedence) && (associativity === argAssociativity) && !assocWithArg) {
+          parens = true;
+        } else if (argPrecedence < precedence) {
+          parens = true;
+        } else {
+          parens = false;
+        }
+
+        parensArray.push(parens);
```
Comment (josdejong) [inline lib/expression/node/OperatorNode.js]: Maybe you can use `var parensArray = args.map(function () {})` instead of a `forEach` and `parensArray.push`?

Comment (FSMaxB) [inline lib/expression/node/OperatorNode.js]: I wasn't too familiar with `Array.prototype.map`. I'll take a look at it tomorrow.

**Comment by josdejong** [comment]
Looks good :+1:  You could replace `forEach` with `map` in a couple of places, feel free to change this later on, I will merge your work now.

I will adjust `add` and `sum` coming week to accept more than two arguments and do a release then.

File: lib/expression/node/OperatorNode.js (hunk 1 above)
Comment (josdejong) [inline lib/expression/node/OperatorNode.js]: you will love it ;)

**Pull request merged**